src/bank.rs

```diff
@@ -39,13 +39,16 @@ use window::WINDOW_SIZE;
 /// but requires clients to update its `last_id` more frequently. Raising the value
 /// lengthens the time a client must wait to be certain a missing transaction will
 /// not be processed by the network.
-pub const MAX_ENTRY_IDS: usize = 1024 * 16;
+pub const MAX_ENTRY_IDS: usize = 1024 * 32;
```

Comment:
I believe that's now the second time this very large data structure doubled in size. Was 32 calculated or simply bumped up to counter blowing out the queue?

Reply:
@garious its because banking stage bench is blowing out the queue.  We register every entry id instead of just the Tick ones.

Reply:
is changing that out of scope for this PR?

Reply:
@rob-solana, bench wont pass without this change.  Changing to Tick based last_id's i think is out of scope